virt/vm_templates/virt-creating-vm-template.adoc

```diff
@@ -4,16 +4,16 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-creating-vm-template
 toc::[]
 
-Using Virtual machines templates is an easy way to create multiple virtual machines 
-with similar configuration. After a template is created, reference the template 
+Virtual machine templates are an easy way to create multiple virtual machines
```

Comment:
Your changes look good, but I have a request... this probably should never have said "easy." We don't want to imply that something is easy in the event that a user has a problem with it. Do you mind rewording this further so that we can avoid that? 

Maybe something like this: "You can use virtual machine templates to create multiple virtual machines that have similar configurations."